packages/@aws-cdk/toolkit/lib/toolkit/toolkit.ts

```diff
@@ -77,9 +84,9 @@ export class Toolkit extends CloudAssemblySourceBuilder implements AsyncDisposab
   public readonly toolkitStackName: string;
 
   /**
-   * @todo should probably be public in one way or the other.
+   * The IoHost of this Toolkit
    */
-  private readonly ioHost: IIoHost;
+  public readonly ioHost: IIoHost;
```

Comment:
This was the simplest thing to do to enable testing and the comment made it seem like it wasn't a total no-no...